scipy/sparse/linalg/_eigen/tests/test_svds.py

```diff
@@ -166,19 +166,19 @@ def test_svds_input_validation_v0_1(self, transpose, n):
         if transpose:
             A = A.T
         k = 2
-        message = "`v0` must have shape"
 
         required_length = (A.shape[0] if self.solver == 'propack'
                            else min(A.shape))
+        message = f"`v0` must have shape ({required_length},)."
         if n != required_length:
-            with pytest.raises(ValueError, match=message):
+            with pytest.raises(ValueError, match=re.escape(message)):
                 svds(A, k=k, v0=v0, solver=self.solver)
 
     def test_svds_input_validation_v0_2(self):
         A = np.ones((10, 10))
         v0 = np.ones((1, 10))
-        message = "`v0` must have shape"
-        with pytest.raises(ValueError, match=message):
+        message = "`v0` must have shape (10,)."
+        with pytest.raises(ValueError, match=re.escape(message)):
```

Comment:
same here, no `re` usage please

Reply:
Just agree you all on what you want - then I can apply the changes.

Reply:
It means we revert the changes to the tests, doesn't it?

Reply:
Yes, we don't really need to adjust the tests for the change. We could also use a raw string here if we really want that.

Reply:
@tupui good call on `rf`.